content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -114,6 +114,35 @@ stages:
 The commands in `dvc metrics` and `dvc plots` help you display and compare
 metrics and plots.
 
+## Plots definitions
+
+The list of plots contains one or more user-defined
+[plots](/doc/command-reference/plots#plots-definitions). Here's an example that
+tells DVC that `auc.json` is viable for visualization:
+
+```yaml
+stages:
+  build:
+    cmd: python train.py
+    deps:
+      - features.csv
+    outs:
+      - model.pt
+      - auc.json
+    metrics:
+      - accuracy.txt:
+          cache: false
+plots:
```

Comment:
are those still supported per stage?

Reply:
You mean `plots outputs`? Yes, they are supported

Reply:
OK but maybe then the `outs` row (**Stage entries** table) should mention these can become viable for visualization via [top-level plot definitions](https://dvc-org-7086-flexible-p-d0hfxv.herokuapp.com/doc/user-guide/project-structure/pipelines-files#top-level-plot-definitions).